src/CMakeLists.txt

```diff
@@ -279,6 +285,14 @@ macro(upm_doxygen)
   endif ()
 endmacro(upm_doxygen)
 
+if (NPM)
+add_custom_target (npmpkg)
```

Comment:
Should be able to merge the add_custom_command into this add_custom_target (w/o the POST_BUILD).